micmac_li3ds/distortion.py

```diff
@@ -5,22 +5,25 @@
 _distortion_data_readers = {}
 
 
-def read_info(mod_unif_node):
+def read_info(calib_disto_node):
     """
-    Read the distortion info from the ``ModUnif`` XML node. Return a tuple
-    of three elements: (type of model, list of states, list of params).
+    Read the distortion info from the ``CalibDistortion`` XML node.
+    Return a tuple of two elements: (type of model, dict parameters).
 
-    :param mod_unif_node: the ``ModUnif`` XML node.
+    :param disto_node: the ``CalibDistortion`` XML node.
     """
-    type_modele_node = xmlutil.child(mod_unif_node, 'TypeModele')
-    typ = type_modele_node.text
-    states, params = _distortion_data_reader(typ)(mod_unif_node)
-    return typ, states, params
+    disto_nodes = xmlutil.children(calib_disto_node, '*')
+    assert(len(disto_nodes) == 1)
```

Comment:
Don't we want to raise a `RuntimeError` or `ValueError` here as we do everywhere else?